moveos/moveos-verifier/src/metadata.rs

```diff
@@ -469,6 +470,116 @@ pub fn is_allowed_input_struct(name: String) -> bool {
+                    | Bytecode::LdU8(_)
+                    | Bytecode::LdU16(_)
+                    | Bytecode::LdU32(_)
+                    | Bytecode::LdU64(_)
+                    | Bytecode::LdU128(_)
+                    | Bytecode::LdU256(_)
+                    | Bytecode::CastU8
+                    | Bytecode::CastU16
+                    | Bytecode::CastU32
+                    | Bytecode::CastU64
+                    | Bytecode::CastU128
+                    | Bytecode::CastU256
+                    | Bytecode::LdConst(_)
+                    | Bytecode::LdTrue
+                    | Bytecode::LdFalse
+                    | Bytecode::CopyLoc(_)
+                    | Bytecode::MoveLoc(_)
+                    | Bytecode::StLoc(_)
+                    | Bytecode::Call(_)
+                    | Bytecode::CallGeneric(_)
+                    | Bytecode::Pack(_)
+                    | Bytecode::PackGeneric(_)
+                    | Bytecode::Unpack(_)
+                    | Bytecode::UnpackGeneric(_)
+                    | Bytecode::ReadRef
+                    | Bytecode::WriteRef
+                    | Bytecode::FreezeRef
+                    | Bytecode::MutBorrowLoc(_)
+                    | Bytecode::ImmBorrowLoc(_)
+                    | Bytecode::MutBorrowField(_)
+                    | Bytecode::MutBorrowFieldGeneric(_)
+                    | Bytecode::ImmBorrowField(_)
+                    | Bytecode::ImmBorrowFieldGeneric(_)
+                    | Bytecode::Add
+                    | Bytecode::Sub
+                    | Bytecode::Mul
+                    | Bytecode::Mod
+                    | Bytecode::Div
+                    | Bytecode::BitOr
+                    | Bytecode::BitAnd
+                    | Bytecode::Xor
+                    | Bytecode::Shl
+                    | Bytecode::Shr
+                    | Bytecode::Or
+                    | Bytecode::And
+                    | Bytecode::Not
+                    | Bytecode::Eq
+                    | Bytecode::Neq
+                    | Bytecode::Lt
+                    | Bytecode::Gt
+                    | Bytecode::Le
+                    | Bytecode::Ge
+                    | Bytecode::Abort
+                    | Bytecode::Nop
+                    | Bytecode::VecPack(_, _)
+                    | Bytecode::VecLen(_)
+                    | Bytecode::VecImmBorrow(_)
+                    | Bytecode::VecMutBorrow(_)
+                    | Bytecode::VecPushBack(_)
+                    | Bytecode::VecPopBack(_)
+                    | Bytecode::VecUnpack(_, _)
+                    | Bytecode::VecSwap(_) => {}
+                }
+            }
+
+            if !invalid_bytecode.is_empty() {
+                let func_name = self
+                    .env
+                    .symbol_pool()
+                    .string(fun.get_name())
+                    .as_str()
+                    .to_string();
+
+                let error_msg = format!(
```

Comment:
If use built-in instruction related to global storage, Is it directly replaced by the new instruction of Storage Abstract, or an error is thrown directly?

Reply:
The following code inserts a compilation error in the environment, and causing the subsequent checking program to halt the compilation process.

```rust
self.env.error(&fun.get_loc(), error_msg.as_str());
```